laserchicken/build_volume.py

```diff
@@ -0,0 +1,40 @@
+import inspect
+
+import laserchicken
+from laserchicken.volume_specification import Volume
+
+
+def create_volume_map():
+    """ Generate map of volume types that are available """
+    volume_map = {}
+    for name, obj in inspect.getmembers(laserchicken.volume_specification):
+        if inspect.isclass(obj) and issubclass(obj, Volume) and obj is not Volume:
+            volume_map[obj.TYPE] = obj
+    return volume_map
+
+
+VOLUMES = create_volume_map()
+
+
+def build_volume(vol_name, *args, **kwargs):
```

Comment:
Don't abbreviate if you don't need to. Cheap way to save a few people from being confused in the future. So 'volume_name'. Or even more descriptive maybe: 'volume_type'.

Reply:
Right, fixed this, thanks!